CMakeLists.txt

```diff
@@ -213,6 +213,10 @@ if(CUDA_FOUND)
     add_subdirectory(src/cudadecoderbin)
 endif()
 
+add_library(kaldi-fst ALIAS kaldi-fstext)

```

Comment:
I don't like adding an alias.  It will be confusing IMO.  Why not explicitly add the dependency that fstbin depends on fstext?

Reply:
I agree, I think the best solution is actually to not generate these `CMakeLists.txt` and add them for versionning so we can explicitly write the dependencies for each target. Is there a problem with this?

Reply:
I think that they wanted the Makefiles to be the source of truth, so that cmake and make don't become out-of-sync. Of course, that is a complicated endeavour.

Reply:
Original design was to checkin all generated `CMakeLists.txt`s and remove the generator afterward, aka, make the cmake build system first class. But it will introduce maintenance burden. So for now, the generator is kept and `CMakeLists.txt`s are generated on the fly.